From eef1675fcb6d896805e2c49a4d4aefd4c584faf6 Mon Sep 17 00:00:00 2001
From: Harini Katakam <harini.katakam@xilinx.com>
Date: Mon, 17 Mar 2014 13:16:44 +0530
Subject: [PATCH 085/509] SPI: cadence: Use one line comments

https://github.com/analogdevicesinc/linux.git xcomm_zynq
commit fe8ed1a6155f7cc93e2eddd902bbbef31f31a95a

Changed multi line comments to one line comments where possible.
Corrected multi line comment in one place.

Signed-off-by: Harini Katakam <harinik@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 drivers/spi/spi-cadence.c |   15 +++++----------
 1 files changed, 5 insertions(+), 10 deletions(-)

diff --git a/drivers/spi/spi-cadence.c b/drivers/spi/spi-cadence.c
index 2573d9c..60dc6d6 100644
--- a/drivers/spi/spi-cadence.c
+++ b/drivers/spi/spi-cadence.c
@@ -25,14 +25,10 @@
 #include <linux/platform_device.h>
 #include <linux/spi/spi.h>
 
-/*
- * Name of this driver
- */
+/* Name of this driver */
 #define CDNS_SPI_NAME		"cdns-spi"
 
-/*
- * Register offset definitions
- */
+/* Register offset definitions */
 #define CDNS_SPI_CR_OFFSET	0x00 /* Configuration  Register, RW */
 #define CDNS_SPI_ISR_OFFSET	0x04 /* Interrupt Status Register, RO */
 #define CDNS_SPI_IER_OFFSET	0x08 /* Interrupt Enable Register, WO */
@@ -359,7 +355,8 @@ static irqreturn_t cdns_spi_irq(int irq, void *dev_id)
 	if (intr_status & CDNS_SPI_IXR_MODF_MASK) {
 		/* Indicate that transfer is completed, the SPI subsystem will
 		 * identify the error as the remaining bytes to be
-		 * transferred is non-zero */
+		 * transferred is non-zero
+		 */
 		cdns_spi_write(xspi->regs + CDNS_SPI_IDR_OFFSET,
 				CDNS_SPI_IXR_DEFAULT_MASK);
 		complete(&xspi->done);
@@ -794,9 +791,7 @@ static struct of_device_id cdns_spi_of_match[] = {
 };
 MODULE_DEVICE_TABLE(of, cdns_spi_of_match);
 
-/*
- * cdns_spi_driver - This structure defines the SPI subsystem platform driver
- */
+/* cdns_spi_driver - This structure defines the SPI subsystem platform driver */
 static struct platform_driver cdns_spi_driver = {
 	.probe	= cdns_spi_probe,
 	.remove	= cdns_spi_remove,
-- 
1.7.5.4

